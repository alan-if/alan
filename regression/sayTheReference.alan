Every r Isa object
  Has n location.
  Description
    Say The n Of this.
End Every r.

The rr Isa r At l
  Has n l.
End The rr.

The l Isa location
End The l.

Start At l.
