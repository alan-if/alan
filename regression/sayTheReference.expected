########## sayTheReference ##########

        No warnings or errors detected.



L
The l

> 

Do you want to UNDO, RESTART, RESTORE or QUIT ? 
